src/components/select/bl-select.css

```diff
@@ -0,0 +1,218 @@
+:host {
+  width: 200px;
+  display: inline-block;
+
+  --bl-select-menu-z-index: 1;
```

Comment:
Do we need to give this property outside? If so, it should be documented in the component file. But I think we should have a standard about z-levels and use them in the components accordingly. The components like tooltip, select, date-picker, dialog, drawer will need a consistent vertical levelling. @buseselvi

Reply:
I think we should give this property outside. This is why I didn't remove z-index property from :host. In some cases it might need to be edited.

Reply:
I would consider to have static value for now and convert it to the level that we’ll define with our design token later. In that case, let’s say we have a `—bl-level-high: 10;` variable and use it here, and user wants to change it, then they can override `—bl-level-high` in whole page or in a scope, but we can still guarantiee that our other components those uses same level will continue to work consistently with select component. But if a user sets the `z-index` of a specific select, then there will be a risk to have conflict with our other popover components like tooltip, date-picker, dialog, drawer etc).

Reply:
Alright, that solution will be better and more efficient. I will remove this value from `:host`